domains/new_dev_project/core/port/outgoing/initial_data.py

```diff
@@ -0,0 +1,10 @@
+from abc import abstractmethod
+
+class InitialDataInterface():    
```

Comment:
Not a valid class name. Class name needs to signify business logic wise what is happening. Initial data does not signify business wise what we want to achieve.